packages/custom-functions-metadata/src/main.ts

```diff
@@ -1,4 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-export * from "./generate";
+export { generateCustomFunctionsMetadata } from "./generateCustomFunctionsMetadata";
```

Comment:
Would prefer to keep `export *` as it was originally.

Reply:
I changed it because it was very difficult to understand what exactly this package was exporting.

This gives a complete list of types that are intended for export.

Reply:
Triage says this is OK, but the change should be done in all packages in order to be consistent in the repo.  It should be done in a separate PR along with the other packages and we should continue to us `export *` for this PR.